policy-controller/src/main.rs

```diff
@@ -174,7 +174,10 @@ async fn main() -> Result<()> {
     // Build the status index which will be used to process updates to policy
     // resources and send to the status controller.
     let (updates_tx, updates_rx) = mpsc::unbounded_channel();
-    let status_index = status::Index::shared(hostname, claims, updates_tx);
+    let status_index = status::Index::shared(hostname.clone(), claims.clone(), updates_tx);
+
+    // Spawn the status controller reconciliation.
+    tokio::spawn(status::Index::run(status_index.clone()));
```

Comment:
nit, take it or leave it: maybe worth `instrument`ing this task (and the `status_controller` task) with a `tracing` span, since the index and controller tasks log similar messages?

Reply:
https://github.com/linkerd/linkerd2/pull/10444/commits/d7f0e70f93bc4cb13f1a2b5d1c7cc99e2184379e. Lmk your thoughts on those span names, but yes probably a good idea to instrument these.

Reply:
seems fine to me --- `status_controller` and `status_index` would also be fine imo.